kwc-picker.js

```diff
@@ -221,6 +242,9 @@ class KwcPicker extends PolymerElement {
             };
         }
     }
+    _computeCheck(key, index) {
+        return key == index ? false : true;
```

Comment:
`return key !== index;` would have the exact same result as a comparison it is evaluated as a boolean